models/mixins/relations/user.coffee

```diff
@@ -21,7 +21,7 @@ module.exports =
       options.data = @pick('email')
       fetch.call account, options
 
-    authentications = new Backbone.Collection
+    authentications = new Backbone.Collection @get 'authentications'
     authentications.url = "#{API_URL}/api/v1/me/authentications"
```

Comment:
It's different in that we don't need to add the extra request b/c we already bootstrapped the data—but one reason this might be useful is that it would allow later interacting with the REST endpoints through `authentications.add(auth).save()` `authentications.first().destroy()`, e.g. if we wanted to POST/DELETE authentications.

(We don't really leverage this today, and I think it is kind of overkill to wrap this in a collection in the first place—but then again I think there's a lot of un-ideal findings about this OO way of wrapping REST that I gripe about above... so I'm being surgical here and doing an MVP fix).